### PR TITLE
[MIG][13.0] stock_picking_report_valued: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
TT21690

### CODE REVIEW

File: stock_picking_report_valued/models/stock_move_line.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Copyright 2014-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2018 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# Copyright 2016-2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
```
Comment (emagdalenaC2i) [inline stock_picking_report_valued/models/stock_move_line.py]:
```suggestion
from odoo import fields, models
```

File: stock_picking_report_valued/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Copyright 2014-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2016 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
```
Comment (emagdalenaC2i) [inline stock_picking_report_valued/models/stock_picking.py]:
```suggestion
from odoo import fields, models
```

File: stock_picking_report_valued/models/stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    def _compute_amount_all(self):
+        """This is computed with sudo for avoiding problems if you don't have
+        access to sales orders (stricter warehouse users, inter-company
+        records...).
+        """
+        for pick in self:
+            round_curr = pick.sale_id.currency_id.round
+            amount_tax = 0.0
+            for tax_id, tax_group in pick.get_taxes_values().items():
```
Comment (emagdalenaC2i) [inline stock_picking_report_valued/models/stock_picking.py]:
```suggestion
            for tax_group in pick.get_taxes_values().items():
```

**Comment by sergio-teruel** [comment]
@emagdalenaC2i Thanks for your review.
when I do a migration review if the title contains [WIP] means that it is not ready to be reviewed.
Now is ready!! jeje
Ahhh changes done!!!

**Comment by emagdalenaC2i** [comment]
ok, I didn't notice the [WIP] text

**Comment by sergio-teruel** [comment]
Yes, I do not know if WIP (Work in progress) is an OCA official tag...

**Comment by emagdalenaC2i** [comment]
No, I know. I mean that I did't see the [WIP] text in the title
But are there any issues with runbot?
The code LGTM but I want to test it

**Comment by sergio-teruel** [comment]
I do not know...

File: stock_picking_report_valued/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  * Oihane Crucelaegui
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Pedro M. Baeza
+  * Antonio Espinosa
+  * Carlos Dauden
+  * David Vidal
+  * Luis M. Ontalba
+  * Ernesto Tejeda
```
Comment (ps-tubtim) [inline stock_picking_report_valued/readme/CONTRIBUTORS.rst:12]: Please add you as contributor.

File: stock_picking_report_valued/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright 2016-2019 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2017 David Vidal - Tecnativa <david.vidal@tecnativa.com>
+# Copyright 2017 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Valued Picking Report",
+    "summary": "Adding Valued Picking on Delivery Slip report",
+    "version": "13.0.1.0.0",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (ps-tubtim) [inline stock_picking_report_valued/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Comment by sergio-teruel** [comment]
@ps-tubtim changes done!!

**Review by ps-tubtim — APPROVED** [review]
Code LGTM

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@sergio-teruel Is this ready ?

**Comment by sergio-teruel** [comment]
Yes.. is ready.. :smile:

**Comment by pedrobaeza** [comment]
Yes, let's go!

/ocabot merge

**Pull request merged**